spec/features/hq/admin_user_authentication_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'AdminUser authentication', :type => :feature do
+
+  scenario 'Restricted pages should be accessible to logged in users' do
+    i_sign_in_as_admin
+    and_i_should_be_on_root_page
+  end
+
+  scenario 'Restricted pages should not be accessible to logged out users' do
+    i_sign_in_as_admin
+    i_sign_out
+    and_i_should_be_on_new_admin_user_session_page
```

Comment:
Minor point: especially in a feature test, it's probably a good idea to use more user-oriented terms. Here, `and_i_should_be_on_sign_in_page` rather than `new_admin_user_session_page`.